lldb/test/API/lang/swift/stdlib/ContiguousArray/main.swift

```diff
@@ -0,0 +1,20 @@
+// main.swift
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+// -----------------------------------------------------------------------------
+class Class {}
+
+func main() {
+	let array = ContiguousArray<Class>([Class()])
+	print(array)// Set breakpoint here
```

Comment:
print generates a lot of code and thus slows down the test, so if you can get away with it, you can also write this as
```
func use<T>(_ t: T) {}
...
use(array)
```